src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbLogger.java

```diff
@@ -204,6 +204,19 @@ private String updateItemRequest(UpdateItemRequest updateRequest) {
                 : "");
     }
 
+    private String deleteItemRequest(DeleteItemRequest deleteItemRequest) {
```

Comment:
deleteItemRequestToString instead? (and generalize to other objects?)

Reply:
I addressed this is in 23cc6edcbe4e12c6b4f35669de8b6d53e6f2ea17 in the fix-conditional-update branch